<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectPage</class>
 <widget class="QWidget" name="ProjectPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>233</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Define Project</string>
  </property>
  <layout class="QGridLayout">
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>This wizard will help you to create a &lt;b&gt;XINX&lt;/b&gt; project and define some important variables.&lt;br/&gt;&lt;br/&gt;
In &lt;b&gt;XINX&lt;/b&gt;, a project is used to use the &lt;i&gt;file research&lt;/i&gt;in a project, use a &lt;i&gt;revision control system&lt;/i&gt;, or use for other purpose in the &lt;b&gt;XINX&lt;/b&gt; modules.&lt;br/&gt;&lt;br/&gt;
A project can define web services too.</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Name of the project :</string>
     </property>
     <property name="buddy">
      <cstring>m_projectNameEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="m_projectNameEdit">
     <property name="whatsThis">
      <string>Name of the project showed in the title bar of &lt;b&gt;XINX&lt;/b&gt;, the project name is appended to the application server directory and showed the project directory.</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>&amp;Project Directory :</string>
     </property>
     <property name="buddy">
      <cstring>m_projectPathEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="DirectoryEdit" name="m_projectPathEdit">
     <property name="whatsThis">
      <string>&lt;p&gt;This is the project directory, generally where is stored the &lt;i&gt;configuration.xml&lt;/i&gt; file.&lt;/p&gt;
&lt;p&gt;This is the racine where the &lt;i&gt;project file view&lt;/i&gt; start.&lt;/p&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>61</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="2">
    <widget class="QToolButton" name="m_projectPathBtn">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../application.qrc">
       <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DirectoryEdit</class>
   <extends>QLineEdit</extends>
   <header>directoryedit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../application.qrc"/>
 </resources>
 <connections/>
</ui>
